### ISSUE TITLE
lime-web-components: Set up some kind of release flow or branches or something that will let us "collect" breaking changes, and release a bunch all at once

### ISSUE THREAD

**Issue by adrianschmidt** [Lundalogik/lime-web-components#264]
We want to avoid making breaking changes to this interface too often, and both @jgroth and I have things we would like to get done "next time we release a new major version", but there's obviously a rather large risk that we won't remember or have time to do those things at that time. So it would be great to be able to do those fixes, and release them all together at a given time.

Maybe all we need is a branch called "next-major" or something… 🤔

**Comment by jgroth** [comment]
Yeah we should probably add a branch with those changes :sweat_smile: It should not be too hard to keep it up to date, this is a quite slow moving repository with small changes. Doesn't semantic release have automatic handling of brances named `dev`/`alpha`/`beta` etc? Then we would get a released package as well making it easier to test the breaking changes in the wild

**Comment by adrianschmidt** [comment]
Yeah, exactly. First step would be to just add a branch, with some branch protections similar to those for `master` (require PR approval etc). A possible second step would be to set up semantic-release to do prereleases.

`alpha` seems like a good choice to start with, I think 👍

**Comment by adrianschmidt** [comment]
Idea: if possible, add a check that disallows merging any commits marked as breaking into `master` from any branch other than `alpha` and `beta`.
